### PR TITLE
Test Kotlin 1.9.0-Beta

### PR DESCRIPTION
**Pull request by eskatos**
See https://github.com/JetBrains/kotlin/releases/tag/v1.9.20-Beta

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test AST

**Comment by eskatos** [comment]
@bot-gradle test AST

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinMultiplatformPluginSmokeTest.groovy [hunk 1]
```diff
@@ -48,7 +48,7 @@ class KotlinMultiplatformPluginSmokeTest extends AbstractKotlinPluginSmokeTest {
         result.task(':tasks').outcome == SUCCESS
 
         where:
-        kotlinVersion << TestedVersions.kotlin.versions
+        kotlinVersion << TestedVersions.kotlin.versions.dropWhile { !it.startsWith('1.9') }
```
Comment (ov7a) [inline subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinMultiplatformPluginSmokeTest.groovy:51]: ❓ This contradicts "Gradle is tested with Kotlin 1.6.10 through 1.9.20-Beta.", right? Is there an alternative for other versions for this test?

Comment (eskatos) [reply subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinMultiplatformPluginSmokeTest.groovy:51]: Good catch. It's a remnant of a local modification. Removed!

**Review by ov7a — APPROVED** [review]

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**